myblog/admin.py

```diff
@@ -1,5 +1,7 @@
-from django.contrib import admin
-from . import models
-# Register your models here.
-
-admin.site.register(models.Post)
\ No newline at end of file
+from django.contrib import admin
+
+from . import models

```

Comment:
O que acha de aqui deixar?
```suggestion
from myblog.models import Post

```
Para la embaixo ficar 

```
admin.site.register(Post)
```